CMakeLists.txt

```diff
@@ -67,428 +67,436 @@ find_package(PkgConfig REQUIRED)
 endif()
 if(UNIX)
-	set(SUNSHINE_ASSETS_DIR "${CMAKE_INSTALL_PREFIX}/${SUNSHINE_ASSETS_DIR}")
+    set(SUNSHINE_ASSETS_DIR "${CMAKE_INSTALL_PREFIX}/${SUNSHINE_ASSETS_DIR}")
 endif()
 
 # use relative assets path for AppImage... maybe for all unix
 if(${SUNSHINE_CONFIGURE_APPIMAGE})
-	string(REPLACE "${CMAKE_INSTALL_PREFIX}" ".${CMAKE_INSTALL_PREFIX}" SUNSHINE_ASSETS_DIR_DEF ${SUNSHINE_ASSETS_DIR})
+    string(REPLACE "${CMAKE_INSTALL_PREFIX}" ".${CMAKE_INSTALL_PREFIX}" SUNSHINE_ASSETS_DIR_DEF ${SUNSHINE_ASSETS_DIR})
 else()
-	set(SUNSHINE_ASSETS_DIR_DEF "${SUNSHINE_ASSETS_DIR}")
+    set(SUNSHINE_ASSETS_DIR_DEF "${SUNSHINE_ASSETS_DIR}")
 endif()
 list(APPEND SUNSHINE_DEFINITIONS SUNSHINE_ASSETS_DIR="${SUNSHINE_ASSETS_DIR_DEF}")
 
 list(APPEND SUNSHINE_EXTERNAL_LIBRARIES
-		libminiupnpc-static
-		${CMAKE_THREAD_LIBS_INIT}
-		enet
-		opus
-		${FFMPEG_LIBRARIES}
-		${Boost_LIBRARIES}
-		${OPENSSL_LIBRARIES}
-		${CURL_LIBRARIES}
-		${PLATFORM_LIBRARIES})
+        libminiupnpc-static
+        ${CMAKE_THREAD_LIBS_INIT}
+        enet
+        opus
+        ${FFMPEG_LIBRARIES}
+        ${Boost_LIBRARIES}
+        ${OPENSSL_LIBRARIES}
+        ${CURL_LIBRARIES}
+        ${PLATFORM_LIBRARIES})
 
 if(NOT WIN32)
-	list(APPEND SUNSHINE_EXTERNAL_LIBRARIES Boost::log)
+    list(APPEND SUNSHINE_EXTERNAL_LIBRARIES Boost::log)
 endif()
 
 add_executable(sunshine ${SUNSHINE_TARGET_FILES})
 
 if(WIN32)
-        set_target_properties(sunshine PROPERTIES LINK_SEARCH_START_STATIC 1)
+    set_target_properties(sunshine PROPERTIES LINK_SEARCH_START_STATIC 1)
 endif()
 
 target_link_libraries(sunshine ${SUNSHINE_EXTERNAL_LIBRARIES} ${EXTRA_LIBS})
 target_compile_definitions(sunshine PUBLIC ${SUNSHINE_DEFINITIONS})
 set_target_properties(sunshine PROPERTIES CXX_STANDARD 17
-                            VERSION ${PROJECT_VERSION}
-                            SOVERSION ${PROJECT_VERSION_MAJOR}
-                            )
+        VERSION ${PROJECT_VERSION}
+        SOVERSION ${PROJECT_VERSION_MAJOR})
 
 if(NOT DEFINED CMAKE_CUDA_STANDARD)
     set(CMAKE_CUDA_STANDARD 17)
     set(CMAKE_CUDA_STANDARD_REQUIRED ON)
 endif()
 
 if(APPLE)
-	target_link_options(sunshine PRIVATE LINKER:-sectcreate,__TEXT,__info_plist,${APPLE_PLIST_FILE})
+    target_link_options(sunshine PRIVATE LINKER:-sectcreate,__TEXT,__info_plist,${APPLE_PLIST_FILE})
 endif()
 
 foreach(flag IN LISTS SUNSHINE_COMPILE_OPTIONS)
-	list(APPEND SUNSHINE_COMPILE_OPTIONS_CUDA "$<$<COMPILE_LANGUAGE:CUDA>:--compiler-options=${flag}>")
+    list(APPEND SUNSHINE_COMPILE_OPTIONS_CUDA "$<$<COMPILE_LANGUAGE:CUDA>:--compiler-options=${flag}>")
 endforeach()
 
-target_compile_options(sunshine PRIVATE $<$<COMPILE_LANGUAGE:CXX>:${SUNSHINE_COMPILE_OPTIONS}>;$<$<COMPILE_LANGUAGE:CUDA>:${SUNSHINE_COMPILE_OPTIONS_CUDA};-std=c++17>)
+target_compile_options(sunshine PRIVATE "$<$<COMPILE_LANGUAGE:CXX>:\
+${SUNSHINE_COMPILE_OPTIONS}>;$<$<COMPILE_LANGUAGE:CUDA>:${SUNSHINE_COMPILE_OPTIONS_CUDA};-std=c++17>")
```

Comment:
This is the only change that feels harder (for me) to parse. Maybe move the `${SUNSHINE_COMPILE_OPTIONS}>;` to the previous line or split both pieces up into separate variables beforehand?

Reply:
I ended up reverting this line and putting an override for now.